pulumi/README.md

```diff
@@ -0,0 +1,81 @@
+This is a typescript [pulumi](https://www.pulumi.com/docs) project. Pulumi is both a command line interface and a set of APIs for building cloud resources with code. This project was created using the `pulumi new AWS-typescript` command. This project is mostly about using pulumi to build things in kubernetes backed by AWS.
+
+# Getting started
+You need both pulumi and the AWS cli installed locally to develop this. If you're on a mac do:
+
+```
+brew install pulumi
+brew install awscli
+```
+
+# Configuring the AWS cli
+Set some environment variables so that the AWS cli can authenticate you.
+```
+export AWS_ACCESS_KEY_ID=<YOUR_ACCESS_KEY_ID> && export AWS_SECRET_ACCESS_KEY=<YOUR_SECRET_ACCESS_KEY>
+```
+
+# Running the code
+
+Clone this repo then run `npm install`. Let's stick to npm for this project and not use yarn.
```

Comment:
I don't think this is a big deal. I think we can stick to version 14 for now or switch to 18 -- whatever works. It's probably best for us all to be developing against the same node version though because of what you're seeing -- subtle changes in the packages will result from running it against a different version. This is easy to setup with nvm. Once you have nvm you can just do `nvm install <version you want>` and then switch into it.

Reply:
Looks like 14 is an LTS version and has support until 2023-04-30. I defer to you on which version to use